connectors/http/src/test/java/quarks/test/connectors/http/HttpTest.java

```diff
@@ -80,9 +154,9 @@ public void testStatusCode() throws Exception {
         
         String url = "http://httpbin.org/status/";
         
+        TStream<Integer> stream = topology.collection(Arrays.asList(200, 404, 202));
```

Comment:
Just to clarify, when I mentioned pulling the declaration of the `stream` outside of the request() invocation, I was only talking about for the samples in the javadoc.  It's OK that test code had it that way.  But it's fine to leave it this way now.  Sorry for the confusion.